cli/JSXV4.md

```diff
@@ -286,7 +286,11 @@ React.createElement(Comp.make, {x, y: 7, ?z})
 
 <Comp x key="7">
 // is transformed to
-React.createElement(Comp.make, Jsx.addKeyProp({x: x}, "7"))
+React.createElement(Comp.make, Jsx.addKeyProp(~key="7", {x: x}))
+
+<Comp x key=?Some("7")>
+// is transformed to
+React.createElement(Comp.make, Jsx.addKeyProp(~key=?Some("7"), {x: x}))
```

Comment:
Isn't it `React.addKeyProp` as per https://github.com/rescript-lang/rescript-react/pull/74?

Reply:
Good catch! I'll fix it.